src/Form/atoms/SliderInput.tsx

```diff
@@ -100,13 +103,22 @@ const thumbLeftPosition = (value: number, min: number, max: number ) => {
   return valueToPercent(value, min, max);
 };
 
+const nearMark = (value: number, marks: ISliderMark[]) : number => {
+
+  let closest = marks.reduce((prev, curr) => {
+    return (Math.abs(curr.value - value) < Math.abs(prev.value - value) ? curr : prev);
+  });
+
+  return closest.value;
+};
+
 /**
  * Rules for a getValidMin
  * Max needs to be bigger than Min
  * if Min is not available and Max is positive default is 0
  * if Min is not available Max is negative min will be reduce by 1
  */
-const getValidMin = (max: number, min?: number) : number => {
+const getValidMin = (max: number, min?: number) : number => { 
```

Comment:
Could you check to make sure your VS Code is set to strip trailing whitespaces please. :)